### DISCUSSION TITLE
manage disk space - no recording for predefined list of species

### DISCUSSION THREAD

**Discussion by pddpauw** [mcguirepr89/BirdNET-Pi#1074] [Ideas]
Hi all,

working on a project to monitor species in a nature reserve.  Idea would be to have this as stand alone unit and review recordings after e.g. 30 days.  

Is there an option to define for which birds you don't want to keep recordings (in my case, this would be some very common birds which have like half the database)?  This would help to reduce (significantly!) the disk space.

Preferably, we can keep the counting of that species, but don't save the audio recording.
I know there is a possible setting to exclude species from the list, but to my understanding, also the counting gets ignored in this case.

What I can do for now, is a daily cronjob to remove the audio files for a certain species, but integration of this functionality would be nice!

**Comment by DemBones78** [comment]
this is kind like my request
i think both high light some possible improvements
https://github.com/mcguirepr89/BirdNET-Pi/discussions/972
I gave yours a +1 please do teh same
